packages/botonic-plugin-contentful/src/contentful/manage/manage-entry.ts

```diff
@@ -170,4 +179,27 @@ export class ManageContentfulEntry {
       throw new ResourceNotFoundCmsException(contentId, e)
     }
   }
+
+  private convertValueType(key: ContentFieldType, field: any): any {
+    if (key === ContentFieldType.BUTTONS_STYLE) {
+      return field ? 'QuickReplies' : 'Buttons'
```

Comment:
are 'QuickReplies' or 'Buttons' stored in some constants to be used instead of literals?
Same for 'Asset', in line 191.